### ISSUE TITLE
SimpleAssembler.cs

### ISSUE THREAD

**Issue by zymex22** [nullre/Project-RimFactory-Revived#40]
Hello i use this on my assemblers when creating new machines
from here:
https://github.com/zymex22/Project-RimFactory-Revived/blob/master/Source/ProjectRimFactory/SAL3/Things/Assemblers/Building_SimpleAssembler.cs

```
<modExtensions>
<li Class="ProjectRimFactory.SAL3.Exposables.AssemblerDefModExtension">
<importRecipesFrom>ElectricStove</importRecipesFrom></li>
</modExtensions>
```

Would it be possible to Change it to import more then one workbench?
so:
```
<lil>ElectricStove</li>
<lil>ElectricSmelter</li>
.....
```

**Comment by nullre** [comment]
fixed in fix_#40 branch.
https://github.com/nullre/Project-RimFactory-Revived/commit/0e27570a60eee1e816bdffccfd1764a67f66d6a9#diff-6ed3271b45bd235e25ddadebf0c4ae3b
https://github.com/nullre/Project-RimFactory-Revived/commit/0e27570a60eee1e816bdffccfd1764a67f66d6a9#diff-e02ec7e368b3e140a50e18976805d157

**Comment by zymex22** [comment]
This works can PR =)
